icu4j/main/tests/core/src/com/ibm/icu/dev/test/text/SpoofCheckerTest.java

```diff
@@ -728,6 +740,13 @@ public void testConfData() {
                     continue;
                 }
 
+                if (from.codePoints().anyMatch(c -> UCharacter.hasBinaryProperty(c, UProperty.DEFAULT_IGNORABLE_CODE_POINT))) {
```

Comment:
I think the following would be faster, not that it matters in a test.

static final UnicodeSet DEFAULT_IGNORABLE_CODE_POINT = new UnicodeSet("\\p{di}");
...
if (DEFAULT_IGNORABLE_CODE_POINT.containsSome(from)) {

Reply:
It does not matter that it would be faster, but it does matter that it would be prettier. Done.